### PR TITLE
Processes List, Solves #369

### PR DESCRIPTION
**Pull request by danloa**
- Added endpoint to return the xml of the bpmn definition of a process.
- Added the process list
- Changes to the designer to load the diagram using the nee endpoint.
- A new and simpler process was added to the seeder. Run php artisan db:seed --class=ProcessSeeder 
**Notes**
- The new process form must be added
- The deactivate button is not functional
- The process is edited when pressing the edit rollover icon

Ref: #369

### CODE REVIEW

File: resources/views/processes/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@extends('layouts.layout', ['title' => __('Processes Management')])
+
+@section('sidebar')
+    @include('layouts.sidebar', ['sidebar'=> Menu::get('sidebar_admin')])
```
Comment (velkymx) [inline resources/views/processes/index.blade.php]: This is the wrong sidebar, but we don't have this one specifically.

File: resources/views/designer/designer.blade.php [hunk 2]
```diff
@@ -1,6 +1,8 @@
 @extends('layouts.layout')
 @section('content')
-    <div id="designer-container"></div>
```
Comment (velkymx) [inline resources/views/designer/designer.blade.php]: Undefined variable: process (View: /home/circleci/builds/feature-369/resources/views/designer/designer.blade.php)

File: resources/views/processes/index.blade.php (hunk 1 above)
Comment (danloa) [reply resources/views/processes/index.blade.php]: we changed to  sidebar_designer

File: resources/views/designer/designer.blade.php (hunk 2 above)
Comment (danloa) [reply resources/views/designer/designer.blade.php]: Now the process is an optional parameter in the controller

**Comment by velkymx** [comment]
Still can't add a start event when I edit a process. Also getting some Vue errors.

image

**Pull request merged**